composer.json

```diff
@@ -69,8 +73,7 @@
         "patches": {
             "drupal/content_lock": {
                 "https://www.drupal.org/project/content_lock/issues/2919019": "https://www.drupal.org/files/issues/content_lock-caching-invalidation-2919019-3.patch",
-                "https://www.drupal.org/project/content_lock/issues/2956625": "https://www.drupal.org/files/issues/2018-03-28/error-on-locked-content-by-deleted-user-2956625.2.patch",
-                "https://www.drupal.org/project/content_lock/issues/2949198": "https://www.drupal.org/files/issues/content_lock-unlocking-already-locked-2949198-9.patch"
+                "https://www.drupal.org/project/content_lock/issues/2949198": "https://git.drupalcode.org/project/content_lock/-/merge_requests/3.patch"
```

Comment:
This patch needs more work, see https://www.drupal.org/pift-ci-job/2078065

Reply:
Fixed - https://www.drupal.org/pift-ci-job/2092927